experimental/dds/tree2/src/test/snapshots/files/complete-3x3.json

```diff
@@ -9,7 +9,7 @@
\"FieldB\",\"fieldKind\":\"Sequence\",\"change\":[{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":84}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-000000000056\",\"sequenceNumber\":-9007199254740906,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"},{\"change\":{\"maxId\":85,\"changes\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"ModularEditBuilder.Generic\",\"change\":[{\"index\":2,\"nodeChange\":{\"fieldChanges\":[{\"fieldKey\":\"FieldB\",\"fieldKind\":\"Sequence\",\"change\":[{\"count\":1},{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":85}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-000000000057\",\"sequenceNumber\":-9007199254740905,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"},{\"change\":{\"maxId\":86,\"changes\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"ModularEditBuilder.Generic\",\"change\":[{\"index\":2,\"nodeChange\":{\"fieldChanges\":[{\"fieldKey\":\"FieldB\",\"fieldKind\":\"Sequence\",\"change\":[{\"count\":2},{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":86}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-000000000058\",\"sequenceNumber\":-9007199254740904,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"},{\"change\":{\"maxId\":87,\"changes\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"ModularEditBuilder.Generic\",\"change\":[{\"index\":2,\"nodeChange\":{\"fieldChanges\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"Sequence\",\"change\":[{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":87}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-000000000059\",\"sequenceNumber\":-9007199254740903,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"},{\"change\":{\"maxId\":88,\"changes\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"ModularEditBuilder.Generic\",\"change\":[{\"index\":2,\"nodeChange\":{\"fieldChanges\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"Sequence\",\"change\":[{\"count\":1},{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":88}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-00000000005a\",\"sequenceNumber\":-9007199254740902,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"},{\"change\":{\"maxId\":89,\"changes\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"ModularEditBuilder.Generic\",\"change\":[{\"index\":2,\"nodeChange\":{\"fieldChanges\":[{\"fieldKey\":\"FieldC\",\"fieldKind\":\"Sequence\",\"change\":[{\"count\":2},{\"type\":\"Insert\",\"count\":1,\"content\":[{\"type\":\"TestLeaf\",\"value\":\"[object Object]\"}],\"cellId\":{\"localId\":89}}]}]}}]}]},\"revision\":\"beefbeef-beef-4000-8000-00000000005b\",\"sequenceNumber\":-9007199254740901,\"sessionId\":\"beefbeef-beef-4000-8000-000000000001\"}],\"branches\":[]}"
```

Comment:
The new occurrences of "[object Object]" in here are concerning. I don't know what format this is supposed to be, but I think its what we store for summaries, and it being lossy (storing "[object Object]" as a string not the values) seems bad.

Reply:
Was bad input data. Fixed.